eth/vm/logic/system.py

```diff
@@ -227,13 +227,14 @@ def get_stack_data(self, computation: BaseComputation) -> CreateOpcodeStackData:
         return CreateOpcodeStackData(endowment, memory_start, memory_length, salt)
 
     def get_gas_cost(self, data: CreateOpcodeStackData) -> int:
-        return constants.GAS_SHA3WORD * ceil32(data.memory_length) // 32
+        return constants.GAS_CREATE + constants.GAS_SHA3WORD * ceil32(data.memory_length) // 32
 
     def generate_contract_address(self,
                                   stack_data: CreateOpcodeStackData,
                                   call_data: bytes,
                                   computation: BaseComputation) -> Address:
 
+        computation.state.account_db.increment_nonce(computation.msg.storage_address)
```

Comment:
You are the man! :+1: I must have missed that detail in the spec. Thanks for helping out with this :pray:

Reply:
Heh. Sorry for making so much noise out of one missing line. ^_^